docs/docs/train-tutorial.md

```diff
@@ -0,0 +1,338 @@
+
+# note this will not run yet because labels are not specified
+train_config = TrainConfig(data_directory='example_vids/')
+```
+
+### 2. Specify your labels
+
+Your labels should be saved in a `.csv` file with columns for filepath and label. For example:
+
+```console
+$ cat example_labels.csv
+filepath,label
+example_vids/eleph.MP4,elephant
+example_vids/leopard.MP4,leopard
+example_vids/blank.MP4,blank
+example_vids/chimp.MP4,chimpanzee_bonobo
+```
+
+Add the path to your labels with `--labels`.  For example, if your videos are in a folder called `example_vids` and your labels are saved in `example_labels.csv`:
+
+```console
+$ zamba train --data-dir example_vids/ --labels example_labels.csv
+```
+
+In Python, the labels are passed in when `TrainConfig` is instantiated. The Python package allows you to pass in labels as either a file path or a pandas dataframe:
+```python
+labels_dataframe = pd.read_csv('example_labels.csv').set_index('filepath')
+train_config = TrainConfig(data_directory='example_vids/', labels=labels_dataframe)
+```
+
+#### Labels `zamba` has seen before
+
+Your labels may be included in the list of [`zamba` class labels](models.md#species-classes) that the provided models are trained to predict. If so, the relevant model that ships with `zamba` will essentially be used as a checkpoint, and model training will resume from that checkpoint.
+
+#### Completely new labels
+
+You can also train a model to predict completely new labels - the world is your oyster! (We'd love to see a model trained to predict oysters.) If this is the case, the model architecture will replace the final [neural network](https://www.youtube.com/watch?v=aircAruvnKk&t=995s) layer with a new head that predicts *your* labels instead of those that ship with `zamba`. [Backpropogation](https://www.youtube.com/watch?v=Ilg3gGewQ5U) will continue from that point with the new head.
+
+### 3. Choose a model for training
+
+If your videos contain species common to central or west Africa, use the [`time_distributed` model](models.md#time-distributed). If they contain species common to western Europe, use the [`european` model](models.md#european). We do not recommend using the [`slowfast` model](models.md#slowfast) for training because it is much more computationally intensive and slower to run.
+
+Add the model name to your command with `--model`. The `time_distributed` model will be used if no model is specified. For example, if you want to continue training the `european` model based on the videos in `example_euro_vids` and the labels in `example_euro_labels.csv`:
+
+```console
+$ zamba train --data-dir example_euro_vids/ --labels example_euro_labels.csv --model european
+```
+
+In Python, model is specified when `TrainConfig` is instantiated:
+
+```python
+train_config = TrainConfig(
+    data_directory="example_euro_vids/",
+    labels="example_euro_labels.csv",
+    model_name="european",
+)
+```
+
+### 4. Specify any additional parameters
```

Comment:
We could have a third page in the "Advanced Options" section on "Common advanced arguments" and just link to that section from here. I like that structure better. thought?

Reply:
I think that sounds great. I like the first sentence beginning with "And there's so much more!" and then rather than "We'll go through..." link to the advanced options page.